src/Components/AddTodo.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+
+function AddTodo({ addTodo, hideAddTodo }) {
+  const [todoInput, setTodoInput] = useState("");
+  const [alarmTime, setAlarmTime] = useState("");
+  const [submitted, setSubmitted] = useState(false);
+  const [alarmTimeError, setAlarmTimeError] = useState("");
+
+  useEffect(() => {
+    if (todoInput || alarmTime) {
+      setSubmitted(false);
+      setAlarmTimeError("");
+    }
+  }, [todoInput, alarmTime]);
+
+  const handleDone = () => {
```

Comment:
improve function name, function name should be explain what that function is for @kaushaki-kumari